source/server/config/stats/dog_statsd.cc

```diff
@@ -0,0 +1,42 @@
+#include "server/config/stats/dog_statsd.h"
+
+#include <string>
+
+#include "envoy/registry/registry.h"
+
+#include "common/config/well_known_names.h"
+#include "common/network/resolver_impl.h"
+#include "common/stats/statsd.h"
+
+#include "api/stats.pb.h"
+#include "api/stats.pb.validate.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+Stats::SinkPtr DogStatsdSinkFactory::createStatsSink(const Protobuf::Message& config,
```

Comment:
Please make sure that the actual registration system has coverage. You could either add this to one of the v2 integration test configurations or add dedicated config loading tests here: https://github.com/envoyproxy/envoy/blob/master/test/server/config/stats/config_test.cc. Note that you can check your coverage in CI by opening up the CI coverage build and looking at the build artifacts in CircleCI.

Reply:
Done fc4efdaed889cf9cca1811d04c926848c83c11d9

>Note that you can check your coverage in CI by opening up the CI coverage build and looking at the build artifacts in CircleCI.

I just didn't know, thanks!